### ISSUE TITLE
aria-selected scrollable Listbox Example does not follow best practice

### ISSUE THREAD

**Issue by Row** [w3c/aria-practices#3175]
The unselected values/options should have aria-selected="false"

https://www.w3.org/WAI/ARIA/apg/patterns/listbox/examples/listbox-scrollable/

According
https://w3c.github.io/aria/#aria-selected 

Value | Description
-- | --
false | The selectable element is not selected.
true | The selectable element is selected.
undefined | The element is not selectable.

**Comment by JAWS-test** [comment]
I agree with you that it is not best practice to omit aria-selected, see https://github.com/w3c/aria/issues/1094

**Comment by mcking65** [comment]
The [comment by JAWS-test above](https://github.com/w3c/aria-practices/issues/3175#issuecomment-2491687296) fully addresses this issue.

In addition, screen reader users can perceive the selected option in the scrollable listbox example because the item that is selected by moving focus maintains "aria-selected="true" on blur.

Therefore, closing this issue as the example implements ARIA as expected for a single select listbox with the particular functionality represented by the example.
